secrets/admiral.yml

```diff
@@ -37,9 +37,17 @@ dev-mode:
     - admiral.port_scan
     - admiral.tester
 
+task_rate_limit: &task_rate_limit
+  # Modify the rate below as needed. Limits are applied per worker, so
+  # "2/h" means limit one worker to 2 tasks per hour. To get the global
+  # rate limit, you must multiply by the number of workers provisioned.
+  rate_limit: "2/h"
```

Comment:
I just thought of one other thing that I'd like to see here.  Can this comment please include a link to the Celery documentation where the rate limit syntax is defined, in case someone wants to use minutes or seconds, rather than hours?  I think [this is the right link](https://docs.celeryq.dev/en/2.4-archived/reference/celery.app.task.html#celery.app.task.BaseTask.rate_limit), but please confirm.

Reply:
See [5875531](https://github.com/cisagov/admiral/pull/37/commits/58755315a87eea9c50ed11c21532e397bea0f762). That is the right link, but I included a reference from the user guide instead because it provides more syntax examples and states that rate limits are applied per-worker.

Reply:
Good call- the link you used is definitely better.  Thanks!